src/utils.ts

```diff
@@ -7,3 +7,13 @@ export function getNumbers(from: number, to: number): number[] {
 
   return numbers;
 }
+
+export function getPaginatedItems(items: string[], perPage: number) {
+  const paginated = [];
+
+  for (let i = 1; i < Math.ceil(items.length / perPage) + 1; i++) {
```

Comment:
The calculation inside the loop condition is being executed on every iteration, which is not efficient. Consider storing the result of `Math.ceil(items.length / perPage)` in a variable before the loop starts to avoid unnecessary calculations.